src/pymortests/base.py

```diff
@@ -92,3 +94,21 @@ def _inner_wrapper(*args, **kwargs):
             return hypothesis.settings(deadline=dl)(func)(*args, **kwargs)
         return _inner_wrapper
     return _outer_wrapper
+
+
+def skip_if_missing(module_name):
+    """Wrapper for requiring certain modules on tests."""
+    def _outer_wrapper(func):
+        @wraps(func)
+        def _inner_wrapper(*args, **kwargs):
+            try:
+                module = importlib.import_module(module_name)
```

Comment:
Maybe rely on the pyMOR config here instead of a failing import? Some packages have a more complex "it's there" condition for us I think.